test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua

```diff
@@ -91,13 +91,27 @@ local requestParams = {
   interactionLayout = "ICON_ONLY"
 }
 
+local requestParams_noVR = {
```

Comment:
requestParams_noVR is not used in script, propose that must be in line 401 .

Reply:
You are correct, I have updated line 401 to match. Thank you!